app/assets/javascripts/admin/templates/user-index.hbs

```diff
@@ -221,7 +221,13 @@
 
   <div class='display-row'>
     <div class='field'>{{i18n 'admin.user.admin'}}</div>
-    <div class='value'>{{model.admin}}</div>
+    <div class='value'>
+      {{#if model.admin}}
```

Comment:
You know what would be better here to avoid repeating yourself? An ember helper that would display the boolean? For example, something like: `{{show-boolean model.admin}}`

Would be a lot more DRY.